services/gitlab.service

```diff
@@ -0,0 +1,37 @@
+# ExperimentalPlatform
+[Unit]
+Description=Gitlab
+After=gitlab-redis.service ldap-protonet.service
```

Comment:
Just realized the ldap-protonet service name. We should have stronger conventions for these things - since the repo is in experimentalplatform/platform-haproxy, I would have expected this service to be called `platform-ldap.service`. Missed the original PR that introduced this